ThumbnailItem/ThumbnailItem.js

```diff
@@ -2,7 +2,11 @@
  * Provides an Agate-themed thumbnail item.
  *
  * @example
- * <ThumbnailItem src="https://dummyimage.com/64/e048e0/0011ff">An image!</ThumbnailItem>
+ * <ThumbnailItem
+ * 		src="https://dummyimage.com/64/e048e0/0011ff"
+ * 		content="Content"
```

Comment:
In FORTECH-37 for agate/ImageItem, the `children` prop is used for the content. I think that it would be better to match APIs among components and among theme libraries. In another theme library, we uses the `children` prop for the Content and `label` for the Sub Content. So my suggestion is...Let's rename `content` and `subContent` as `children` and `label`.